pnpm-lock.yaml

```diff
@@ -26454,6 +26455,29 @@ packages:
     resolution: {integrity: sha512-dLitG79d+GV1Nb/VYcCDFivJeK1hiukt9QjRNVOsUtTy1rR1YJsmpGGTZ3qJos+uw7WmWF4wUwBd9jxjocFC2w==}
     engines: {node: ^12.17.0 || ^14.13 || >=16.0.0}
 
+  /change-case/3.1.0:
```

Comment:
this package is at version 5.4.4, kinda sad to pick up a version already 2 majors out of date.

Reply:
It went ESM-only, and using such packages in build-cli has proven to be very hard. No matter what I do, TypeScript continues to produce a require. I haven't tried the CJS re-export trick yet though - I forgot about it when I added the change-case dep. I'll try it in a separate PR.